### ISSUE TITLE
Display Walk-Ins Accepted and Business Hours Notes to Food Seekers

### ISSUE THREAD

**Issue by entrotech** [hackforla/food-oasis#1068]
### Overview
Issue #904 identified changes to make it easier for Food Seekers to determine if appointments are required, and see if there are any special notes about the business hours. This issue is to display this information to Food Seekers on the StakeholderDetails component.

### Action Items
- [x] Work with UI/UX to determine how to display the Walk-Ins boolean settings to Food Seekers.
- [x] Work with UI/UX to determine how to display the business hours notes.
- [x] Implement the above on the StakeholderDetails component.
- [x] Determine if these bits of information should also be displayed on the StakeholderPreview component.
- [x] Implement changes to the StakeholderPreview component, if desired.

### Resources/Instructions

Depends on Issue #1067

**Comment by entrotech** [comment]
@fancyham @gigicobos @Benbaillou Should any of the appointment info appear on the Organization Preview pane?

![image](https://user-images.githubusercontent.com/9939032/142924934-b4f26498-aa21-4c31-8095-6a219f4a2681.png)

In addition to the Organization Details pane:

![image](https://user-images.githubusercontent.com/9939032/142925422-022e70a1-f94a-485f-aff8-012000464665.png)

**Comment by Benbaillou** [comment]
From my point of view it could be a good idea to display this information here.

**Question: Automatisation** 
Have we considered to automate the entry by enabling the Data Entry Volunteers to tick a box on their panel to display if an organisation does "walk only", "appointment only"

**Comment by Benbaillou** [comment]
@entrotech - I just wanted to have some updates on the status of this issue and the followings ones:

(https://github.com/hackforla/food-oasis/issues/1041)
https://github.com/hackforla/food-oasis/issues/1042)
(https://github.com/hackforla/food-oasis/issues/1066)
(https://github.com/hackforla/food-oasis/issues/1067)
(https://github.com/hackforla/food-oasis/issues/1068)

They are labeled closed - but, tell if I am wrong, I cannot see the implementation of those features in the Data Entry Volunteer's interface.  Can you explain me why ?

Thanks

**Comment by Benbaillou** [comment]
After a quick test on a wireframe, it looks it could take too much space on the panel.

![Capture d’écran 2021-12-06 à 15 45 45](https://user-images.githubusercontent.com/91577998/144876846-2be5456c-9cec-4c89-b2f6-013de2991cef.png)

@souronion and @fancyham - what are your thoughts?

My thoughts would be not to display this details on the current panel to avoid reading confusion.

**Comment by Benbaillou** [comment]
![Capture d’écran 2021-12-06 à 17 53 19](https://user-images.githubusercontent.com/91577998/144897063-2e5f1574-3867-4ea0-97ab-5393b3a2aa6c.png)
Or will it be possible to do a "Google Maps" feature like that

**Comment by fancyham** [comment]
@Benbaillou I think putting the hours on the org page might be worth considering — perhaps an abbreviated version like:  
Today 3-5pm v

(Similar to Google or Apple Maps.  The v is a disclosure triangle.)
and click on it to get the full listing to see the week’s hours.

**Comment by Benbaillou** [comment]
@fancyham
I just remember we have already a feature that displays if an organisation is open and when it will be closed (see below) => which helps food seekers not to waste time going to the org (one of the goal of this feature)


![Capture d’écran 2021-12-08 à 17 23 23](https://user-images.githubusercontent.com/91577998/145255059-09aea15e-937b-4e5b-bda9-8020daa154fc.png)

**My thoughts** 
1) Along this feature 'open' and 'closing hours' we could add a 'walk-in' box like


![Capture d’écran 2021-12-08 à 17 38 57](https://user-images.githubusercontent.com/91577998/145256592-e62390e0-32b0-465e-b70c-c768ab341da1.png)


**Quick question** @entrotech => is this feasible considering we will have a "Walk-ins checkbox" on the Volunteer interface? can we use this data format?

2) It seems that adding the hours could be an add-on to this feature by displaying the hours of the day and by giving a call-to-action to look for more details. However we already have the button "details" on the preview pane => won't it bring some confusion for the user to have 2 buttons to look for details?
Futhermore, notes regarding caveats to the hours won't be displayed - which was a the goal of the feature.

![Capture d’écran 2021-12-08 à 17 51 11](https://user-images.githubusercontent.com/91577998/145258356-61fb4493-4146-4511-9a08-e02546ed9dc4.png)

@fancyham let me know if this is how you were considering this


-----------

Here are some screenshots from Google Maps to illustrate your wording "Today 3-5pm v "
"v" panel closed
![Capture d’écran 2021-12-08 à 17 12 35](https://user-images.githubusercontent.com/91577998/145252980-333346b7-197f-4283-9561-39cd3adf0f9d.png)

"^" panel opened
![Capture d’écran 2021-12-08 à 17 12 41](https://user-images.githubusercontent.com/91577998/145253002-68d0e547-bd62-4bb1-ac06-b29f89a03fba.png)

**Comment by Benbaillou** [comment]
note for myself :  define walk-ins

**Comment by entrotech** [comment]
@fancyham Is there anything left to do on this? Can we create development issues if there are.

**Comment by entrotech** [comment]
@fancyham Can we close this issue?

**Comment by fancyham** [comment]
@entrotech Is there a listing/URL/search term with a working version of this that I could take a look at?

**Comment by entrotech** [comment]
We have not created the "tag" indicating that walk-ins are welcome yet.  This should be assigned to a developer.

**Comment by fancyham** [comment]
Let us know if there’s any stuff design can help with on that!

**Comment by entrotech** [comment]
I think we can work on adding the tag "Walk-ins Welcome" suggested by @Benbaillou  on Dec 8, 2021. Then that should finish this issue.

There doesn't seem to be an agreement about showing the hours on the list, so we'll hold off on that unless/until we come to an agreement, and then we would create a new issue.
